mne/io/edf/edf.py

```diff
@@ -185,6 +189,9 @@ class RawGDF(BaseRaw):
     exclude : list of str
         Channel names to exclude. This can help when reading data with
         different sampling rates to avoid unnecessary resampling.
+    include : list of str | str
+        Channel names to be picked. Channels in include are picked even if
+        they are in exclude.
```

Comment:
to me it's dangerous to allow include to overwrite the exclude. I would raise a ValueError if the intersection of include and exclude is not empty.

Reply:
Updated to raise a VlueError if both `include` and `exclude` are non-empty values.
This was implemented in `_find_exclude_idx` function.